unst_msh_gen/ocn_ww3.py

```diff
@@ -78,11 +83,11 @@ def create_siz():
 
 #-- create mesh spacing function for the globe
 
-    hmax = 50.0 # maximum spacing [km]
-    hshr = 50  # shoreline spacing
+    hmax = 100.0 # maximum spacing [km]
+    hshr = 100  # shoreline spacing
     nwav = 400.  # number of cells per sqrt(g*H)
-    hmin = 50.0  # minimum spacing
-    dhdx = 0.1  # allowable spacing gradient
+    hmin = 100.0  # minimum spacing
+    dhdx = 0.05  # allowable spacing gradient
```

Comment:
Additional information about how this variable relates to mesh quality would be very helpful to users.  

A note if/how these variables are used if you have an input from a mask file would be appreciated.